.github/workflows/rust-apply-patch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+. .github/workflows/common.sh
+
+checkout_branches "rust-${VERSION_NEW}"
+
+pushd "${SDK_OUTER_SRCDIR}/third_party/coreos-overlay" >/dev/null || exit
+
+updateNeeded=1
+VERSION_OLD=$(sed -n "s/^DIST rustc-\(1.[0-9]*.[0-9]*\).*/\1/p" dev-lang/rust/Manifest | sort -ruV | head -n1)
+[[ "${VERSION_NEW}" = "${VERSION_OLD}" ]] && echo "already the latest Rust, nothing to do" && updateNeeded=0 && exit
+
+pushd "dev-lang/rust" >/dev/null || exit
+git mv $(ls -1 rust-${VERSION_OLD}*.ebuild | sort -ruV | head -n1) "rust-${VERSION_NEW}.ebuild"
+popd >/dev/null || exit
+
+popd >/dev/null || exit
+
+generate_patches dev-lang rust Rust
+
+apply_patches
+
+echo ::set-output name=VERSION_OLD::"${VERSION_OLD}"
+echo ::set-output name=UPDATE_NEEDED::"${updateNeeded}"
```

Comment:
The other scripts don't have this last line. I see that only the ymls for rust are using this variable for conditionally creating the patches... Is there a reason why the others don't do this?

Reply:
Right. Only rust has `UPDATE_NEEDED`.
There is no specific reason for that. I have only realized we should check if the update is actually needed, so added it only to Rust.

Other actions do not have it. But it has not been an issue, because so far `peter-evans/create-pull-request` simply succeeds, even if no update is needed.

However, it is different in case of Rust, which triggers both `create-pull-request` and `repository-dispatch`. The former will anyway succeed. But the latter will create an unnecessary repository dispatch event, which will create another round of actions on the side of portage-stable. That could really look confusing.